source/_integrations/huum.markdown

```diff
@@ -0,0 +1,26 @@
+---
+title: Huum
+description: Instructions on how to integrate a Huum saunas into Home Assistant.
+ha_category:
+  - Climate
+ha_release: 2022.6
+ha_iot_class: Cloud Polling
+ha_codeowners:
+  - '@frwickst'
+ha_domain: huum
+ha_config_flow: true
+ha_platforms:
+  - climate
+ha_integration_type: integration
+---
+
+Integrates Huum saunas into Home Assistant.
+
+The sauna can only be controlled through Huum’s Cloud API; no local connection is available.
+
+You’ll need your username (which is usually your email) and password for the integration.
```

Comment:
```suggestion
You’ll need your username (which is usually your email) and password to configure the integration.
```
Are there any other prerequisites that are worth mentioning?

Reply:
changed